spec/system/assignment_spec.rb

```diff
@@ -5,9 +5,11 @@
 describe "Assignments", type: :system do
   before do
     @primary_mentor = FactoryBot.create(:user)
+    @mentor = FactoryBot.create(:user)
```

Comment:
**Suggestion** Shall we move the instance variable definitions to let block instead ?https://github.com/rubocop/rspec-style-guide#instance-variables